src/aus/sourcepoint.test.js

```diff
@@ -44,7 +43,7 @@ describe('Sourcepoint CCPA', () => {
 	it('points at a real file', (done) => {
 		init();
 		const src = document
-			?.getElementById('sourcepoint-aus-lib')
+			.getElementById('sourcepoint-aus-lib')
```

Comment:
Why is this gone?

Reply:
i guess it was not necesary, `document` must be defined in jest types. it was removed by a fixable rule